nginx/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN chown -R skpr:skpr /var/lib/nginx \
 # directive from above.
 VOLUME /run/nginx
 VOLUME /var/tmp/nginx
+VOLUME /var/lib/nginx/logs
```

Comment:
This fixes an "nginx on boot" error, it wants to write to this file before it loads the nginx.conf.